woocommerce/woocommerce
# Update states.php -> added Senegal Regions

### All Submissions:






### Changes proposed in this Pull Request:
This change is adding the regions of Senegal to woocommerce. 

Closes # .

### How to test the changes in this Pull Request:

1.Switch to this PR branch.
2.Verify state list is available on Checkout page after selecting Senegal as shipping destination Country.


### Other information:




### Changelog entry

> This pull request add The regions of Senegal to the state database
